### PR TITLE
add prettier support for graphql

### PR DESCRIPTION
**Pull request by butlerx**

### CODE REVIEW

File: doc/ale-graphql.txt [hunk 1]
```diff
@@ -13,5 +13,10 @@ You will need the GraphQL ESLint plugin installed for this to work.
 ===============================================================================
 gqlint                                                     *ale-graphql-gqlint*
 
+===============================================================================
+prettier                                                   *ale-graphql-prettier*
```
Comment (w0rp) [inline doc/ale-graphql.txt]: Align the tag to the right margin.

File: doc/ale-graphql.txt [hunk 2]
```diff
@@ -13,5 +13,10 @@ You will need the GraphQL ESLint plugin installed for this to work.
...
+
+See |ale-graphql-prettier| for information about the available options.
```
Comment (w0rp) [inline doc/ale-graphql.txt]: You might want to write `ale-javascript-prettier` here so this sends you to the documentation for the `prettier` fixer.

**Comment by butlerx** [comment]
changes made

**Comment by w0rp** [comment]
Cheers! :beers:

**Pull request merged**